plugin/src/main/kotlin/tanvd/grazi/ide/fus/GraziFUStateCollector.kt

```diff
@@ -0,0 +1,36 @@
+package tanvd.grazi.ide.fus
+
+import com.intellij.internal.statistic.beans.*
+import com.intellij.internal.statistic.eventLog.FeatureUsageData
+import com.intellij.internal.statistic.service.fus.collectors.ApplicationUsagesCollector
+import tanvd.grazi.GraziConfig
+
+@Suppress("MissingRecentApi")
+class GraziFUStateCollector : ApplicationUsagesCollector() {
+    override fun getGroupId(): String = "grazi"
+    override fun getVersion(): Int = 1
+
+    override fun getMetrics(): MutableSet<MetricEvent> {
+        val metrics = HashSet<MetricEvent>()
+
+        val state = GraziConfig.get()
+        val default = GraziConfig.State()
+
+        state.enabledLanguages.forEach {
+            metrics.add(newMetric("language", it.shortCode))
+        }
+
+        addIfDiffers(metrics, state, default, { s -> s.nativeLanguage.shortCode }, "native")
```

Comment:
I would prefer more descriptive ids here, e.g. `native.language`. Currently the meaning of the metric is not clear from event id.